_clients/data-prepper/pipelines.md

```diff
@@ -204,6 +204,49 @@ metrics-pipeline:
       password: admin
 ```
 
+### S3 Log Ingestion Pipeline
+
+The following example demonstrates how to use the S3 source and Grok prepper plugins to process unstructured log data
```

Comment:
Capitalize "Source". Should "prepper" also be capitalized as the name of a plugin? It also looks like there are some line break issues in this paragraph.

Reply:
These line breaks are not rendered in the markdown conversion.